log-collector-script/linux/eks-log-collector.sh

```diff
@@ -509,6 +510,18 @@ get_sysctls_info() {
   ok
 }
 
+get_network_policy_ebpf_info() {
+  try "collect network policy ebpf loaded data"
+  echo "*** EBPF loaded data ***" >> "${COLLECT_DIR}"/networking/ebpf-data.txt
+  LOADED_EBPF=$(/opt/cni/bin/aws-eks-na-cli ebpf loaded-ebpfdata | tee -a "${COLLECT_DIR}"/networking/ebpf-data.txt)
```

Comment:
Two questions:
1. What happens if this is run on an older release where `/opt/cni/bin/aws-eks-na-cli` is not present?
2. Is there any scenario under which `aws-eks-na-cli ebpf` can crash and prevent the log collection script from completing? Do we have/need any guards?